### PR TITLE
New package: GovernmentOfThePRC.ShangHai.PuDong.ZhiHuiPuDong version 3.8.0

### PR DESCRIPTION
**Pull request by DuckDuckStudio**
###### Microsoft Reviewers: [Open in CodeFlow](https://microsoft.github.io/open-pr/?codeflow=https://github.com/microsoft/winget-pkgs/pull/199049)

### CODE REVIEW

File: manifests/g/GovernmentOfThePRC/ShangHai/PuDong/ZhiHuiPuDong/3.8.0/GovernmentOfThePRC.ShangHai.PuDong.ZhiHuiPuDong.locale.zh-CN.yaml [hunk 1]
```diff
@@ -0,0 +1,21 @@
+# Created with komac v2.8.0
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.defaultLocale.1.9.0.schema.json
+
+PackageIdentifier: GovernmentOfThePRC.ShangHai.PuDong.ZhiHuiPuDong
+PackageVersion: 3.8.0
+PackageLocale: zh-CN
+Publisher: 上海市浦东新区人民政府
+PublisherUrl: https://www.pudong.gov.cn/
```
Comment (DuckDuckStudio) [inline manifests/g/GovernmentOfThePRC/ShangHai/PuDong/ZhiHuiPuDong/3.8.0/GovernmentOfThePRC.ShangHai.PuDong.ZhiHuiPuDong.locale.zh-CN.yaml:8]: image

Comment (DuckDuckStudio) [inline manifests/g/GovernmentOfThePRC/ShangHai/PuDong/ZhiHuiPuDong/3.8.0/GovernmentOfThePRC.ShangHai.PuDong.ZhiHuiPuDong.locale.zh-CN.yaml:8]:
```suggestion
PublisherUrl: https://english.pudong.gov.cn/
# https://www.pudong.gov.cn/ - 中文页面在 winget-bot 的验证失败：PreconditionFailed
# 请见：https://github.com/microsoft/winget-pkgs/pull/199049#issuecomment-2543861326
```

image

Can this be waived?

File: manifests/g/GovernmentOfThePRC/ShangHai/PuDong/ZhiHuiPuDong/3.8.0/GovernmentOfThePRC.ShangHai.PuDong.ZhiHuiPuDong.installer.yaml [hunk 2]
```diff
@@ -0,0 +1,21 @@
...
+
+PackageIdentifier: GovernmentOfThePRC.ShangHai.PuDong.ZhiHuiPuDong
+PackageVersion: 3.8.0
+InstallerType: nullsoft
+InstallModes:
+- interactive
+- silent
+- silentWithProgress
+UpgradeBehavior: install
+ReleaseDate: 2024-02-02
```
Comment (DuckDuckStudio) [inline manifests/g/GovernmentOfThePRC/ShangHai/PuDong/ZhiHuiPuDong/3.8.0/GovernmentOfThePRC.ShangHai.PuDong.ZhiHuiPuDong.installer.yaml:12]:
```suggestion
ReleaseDate: 2024-02-02
InstallerSuccessCodes:
  - 1223
```

**Comment by stephengillie** [comment]
Verify:
image
The application appears to require login before displaying the version number.
image

**Review by stephengillie — APPROVED** [review]

**Comment by DuckDuckStudio** [comment]
Sorry for not updating the version number before you approved this pull request...

**Comment by stephengillie** [comment]
Apologies - I was optimizing for availiability.

**Pull request merged**